tests/baselines/reference/importCallExpressionAsyncES3AMD.js

```diff
@@ -139,16 +139,17 @@ define(["require", "exports"], function (require, exports) {
         return cl2;
     }());
     exports.cl2 = cl2;
-    exports.l = function () { return __awaiter(void 0, void 0, void 0, function () {
+    var l = function () { return __awaiter(void 0, void 0, void 0, function () {
         var req;
         return __generator(this, function (_a) {
             switch (_a.label) {
-                case 0: return [4 /*yield*/, new Promise(function (resolve_5, reject_5) { require(['./test'], resolve_5, reject_5); })]; // FIVE
+                case 0: return [4 /*yield*/, import('./test')]; // FIVE

```

Comment:
I didn't notice this before, but doesn't look good.

Reply:
possibly missing a call to `visitNode` somewhere.

Reply:
I fixed this with: https://github.com/microsoft/TypeScript/pull/39820/commits/984e3049a60aa6aaf6dece226c2e7f439e875242

Reply:
but yes, a call to `visitNode` was missing :)